Files/Strings/en-US/Resources.resw

```diff
@@ -1812,4 +1812,10 @@
   <data name="CloudDriveSyncStatus_Unknown" xml:space="preserve">
     <value>Unknown</value>
   </data>
+  <data name="StatusBarControlMultiselect.Text" xml:space="preserve">
+    <value>Multiselect</value>
+  </data>
+  <data name="SettingsShowMultiselectOption.Header" xml:space="preserve">
+    <value>Enable multiselect options</value>
```

Comment:
What about to specify, that for the moment Multiselect is working only in GridView mode?

Reply:
@tsvietOK It's under the experimental features so they don't really need any extra information, but it can't hurt.